Foundation/src/Process_UNIX.cpp

```diff
@@ -182,76 +182,83 @@ ProcessHandleImpl* ProcessImpl::launchByForkExecImpl(const std::string& command,
 	// We therefore limit the maximum number of file descriptors we close.
 	const long CLOSE_FD_MAX = 100000;
 
-	// We must not allocated memory after fork(),
-	// therefore allocate all required buffers first.
-	std::vector<char> envChars = getEnvironmentVariablesBuffer(env);
-	std::vector<char*> argv(args.size() + 2);
-	int i = 0;
-	argv[i++] = const_cast<char*>(command.c_str());
-	for (const auto& a: args)
+	do
 	{
-		argv[i++] = const_cast<char*>(a.c_str());
-	}
-	argv[i] = NULL;
-
-	const char* pInitialDirectory = initialDirectory.empty() ? 0 : initialDirectory.c_str();
+		// We must not allocated memory after fork(),
```

Comment:
spelling: alllocate~~d~~

Reply:
Fixed in [8c4d83c](https://github.com/pocoproject/poco/pull/4267/commits/8c4d83c86860d1730a71d29b58aae1e9d8080db0)